Microsoft.Health.Fhir.sln

```diff
@@ -130,6 +130,14 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.R5.Te
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Health.Fhir.R5.Tests.Integration", "test\Microsoft.Health.Fhir.R5.Tests.Integration\Microsoft.Health.Fhir.R5.Tests.Integration.csproj", "{E18E488B-EE9C-4735-81B1-65C74DAE83CF}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Health.Fhir.R4.Client", "src\Microsoft.Health.Fhir.R4.Client\Microsoft.Health.Fhir.R4.Client.csproj", "{C65993F8-E77B-4F7A-BE49-5CAE9DCA3162}"
```

Comment:
Move these into a separate folder within the solution structure. Maybe `client` or something similar.